### DISCUSSION TITLE
About next release ETA.

### DISCUSSION THREAD

**Discussion by Chanki-Min** [software-mansion/react-native-svg#2592] [Q&A] [answered]
Hello contributers. first i want to give huge thanks about maintaining this library.

Our team need patch made by https://github.com/software-mansion/react-native-svg/pull/2573. it seems like release is bit delayed. is there eta about next release (15.10.2 maybe?)

**Comment by jakex7** [comment, marked as answer]
Hi! The patch from #2573 will be included in the next release `15.11.0` that is expected later this week
